server/routes/partners.js

```diff
@@ -0,0 +1,28 @@
+const validPartners = {
+	labs : 'labs.ft.com'
```

Comment:
Yeah, something like ft.com/nikkei, but until we find out whether these pages exist/get a list for them, for now, this should suffice.